plugins/ui/nvim-highlight-colors.nix

```
@@ -0,0 +1,96 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+
+let
+  inherit (helpers.defaultNullOpts)
+    mkBool
+    mkEnum
+    mkListOf
+    mkStr
+    mkStr'
+    ;
```

To match the style used elsewhere in nixvim:
```suggestion
  inherit (lib.nixvim) defaultNullOpts;
```